plugins/notifiers/client.go

```diff
@@ -21,7 +21,7 @@ type Config struct {
 	Provider string `mapstructure:"provider" validate:"omitempty,oneof=slack"`
 
 	// slack
-	AccessToken string `mapstructure:"access_token" validate:"required_if=Provider slack"`
```

Comment:
can we keep this access token to avoid breaking changes

Reply:
added changes to maintain backward compatibility and additional test cases for the same